app/javascript/components/Counter.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from "react";
+import { hot } from "react-hot-loader/root";
+
+function Counter() {
+  const [count, setCount] = useState(0);
+
+  return (
+    <>
+      <p>You clicked {count} times!</p>
+      <button onClick={() => setCount(count + 1)}>Click me</button>
+    </>
+  );
+}
+
+export default hot(Counter);
```

Comment:
I'd rather not have example code like this, we should be able to put a snippet in the readme or do a link to a gist or something so we don't have to remember to remove this on every new application

Reply:
@hiattp just wanted an example showing a HMR-ready export, though on a second thought, it's enough to mention in readme 👍

Reply:
@hiattp updated